docs/tutorial/CustomDataLoaderOnSL.ipynb

```diff
@@ -923,18 +905,15 @@
    ]
   },
   {
-   "attachments": {},
    "cell_type": "markdown",
-   "id": "2ef012c0",
    "metadata": {},
    "source": [
-    "### 2. 将单方的DataBuilder 包装(wrap)起来"
```

Comment:
1. please keep the numeric sequence number correspondence the same.
2. `wrap`->`Wrap`